packages/braid-design-system/src/lib/components/Toggle/Toggle.screenshots.tsx

```diff
@@ -188,7 +188,36 @@ export const screenshots: ComponentScreenshot = {
         />
       ),
     },
-
+    {
+      label: 'With Inline text (standard size)',
+      Container: ({ children }) => (
+        <div style={{ maxWidth: '300px' }}>{children}</div>
+      ),
+      Example: ({ id, handler }) => (
+        <Inline space="xsmall" alignY="center">
+          <Toggle on={true} label="Toggle" id={id} onChange={handler} />
+          <Text>Inline text</Text>
+        </Inline>
+      ),
+    },
+    {
+      label: 'With Inline text (small size)',
+      Container: ({ children }) => (
+        <div style={{ maxWidth: '300px' }}>{children}</div>
+      ),
+      Example: ({ id, handler }) => (
+        <Inline space="xsmall" alignY="center">
+          <Toggle
+            on={true}
+            label="Toggle"
+            id={id}
+            onChange={handler}
+            size="small"
+          />
+          <Text size="small">Inline text</Text>
+        </Inline>
+      ),
+    },
```

Comment:
These screenshot tests don't look like how I'd expect them to look.

<img width="800" alt="image" src="https://github.com/seek-oss/braid-design-system/assets/5663042/8515e724-1763-4599-b123-0d73733ccd6b">

I would've though the small toggle screenshot above would've changed, so there shouldn't be a need to add these new ones.

Reply:
![image](https://github.com/seek-oss/braid-design-system/assets/33821218/6a21a03a-9c97-4a04-ac18-145049bfe2f8)
Hmm, I'm not seeing the same results
Yep the small toggle screenshot has also changed slightly, but even checking that this alignment issue was still missed. It might be worth adding this test because it's easy to see if the text here is misaligned. With the `Toggle` on its own, it can be hard to notice the text being slightly off centre if you aren't paying close attention. Thoughts?

Reply:
Weird. Maybe it was because I was looking in the chromatic storybook specifically.

It _is_ hard to spot, but IMO one of the main values of these screenshots is catching regressions, rather than being a source of truth for layout or to show usage examples. The small toggle screenshot will catch any regressions in this alignment in the future, so IMO these extras aren't that useful.

Reply:
True, but I suspect this came about when adding the SEEK Jobs theme as the error is not in older themes. I guess having this test makes it easier to ensure alignment if a new theme is added in future. What do you think?